src/libraries/Native/AnyOS/System.Globalization.Native/pal_calendarData.h

```diff
@@ -7,7 +7,7 @@
 #include <stdlib.h>
 
 #include "pal_locale.h"
-#include "pal_compiler.h"
+#include "compiler.h"
```

Comment:
Why was this renamed?

Reply:
I renamed it since I moved it to the AnyOS/Common directory, and I understood that the “pal” prefix was for Unix related pals. Like the ones here: https://github.com/dotnet/runtime/tree/master/src/libraries/Native/AnyOS/brotli/common

Reply:
brotli/common is a verbatim copy of the brotli library. We did not invent those file names.

I do not think that the split between Unix/Windows/AnyOS is very useful. CoreCLR has number of Windows or Unix specific parts and we do not have them under Unix/Windows sub-directories either. We did not have these sub-directories originally. They were introduced in https://github.com/dotnet/corefx/pull/10231 for reasons that I do not understand.

Reply:
I see. Maybe that cleanup can be made later?

Then you would suggest I just move the compiler header to AnyOS/Common at the moment but preserve the pal_compiler.h name?

Reply:
Yes, that would be fine with me. Or do not move anything in this PR, and just get of the extra AnyOS/Unix/Windows directory layer (in separate PR).